Scripts/SaneDocReport/SaneDocReport.yml

```diff
@@ -2,27 +2,39 @@ args:
 - default: true
   description: sane report json file contents base64
   isArray: false
-  name: "sane_docx_report_base64"
+  name: sane_docx_report_base64
   required: true
   secret: false
+- default: false
+  description: orientation of the report
+  isArray: false
+  name: orientation
+  required: false
+  secret: false
+- default: false
+  description: the paper size of the report
+  isArray: false
+  name: paperSize
```

Comment:
```suggestion
  name: paper_size
```

Reply:
I can't change this, this is information that is passed from Golang, @Bars92 can elaborate.